### PR TITLE
Reddit Rework!

### PR DESCRIPTION
**Pull request by pedrofracassi**
## What does this PR do?
- Unifies the reddit command codebase into `RandomRedditPostCommand`. No need to write the same run function anymore!
- Moves all reddit commands to `/reddit`

### CODE REVIEW

File: src/commands/reddit/aww.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const RandomRedditImageCommand = require('../../structures/command/RandomRedditPostCommand.js')
```
Comment (bolsomito) [inline src/commands/reddit/aww.js]: ```const { RandomRedditImageCommand } = require('../../')```

File: src/commands/reddit/copypasta.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const RandomRedditPostCommand = require('../../structures/command/RandomRedditPostCommand.js')
```
Comment (bolsomito) [inline src/commands/reddit/copypasta.js]: ```const { RandomRedditPostCommand } = require('../../')```

File: src/commands/reddit/hmmm.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const RandomRedditPostCommand = require('../../structures/command/RandomRedditPostCommand.js')
```
Comment (bolsomito) [inline src/commands/reddit/hmmm.js]: ```const { RandomRedditPostCommand } = require('../../')```

File: src/commands/reddit/oddlysatisfying.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+const RandomRedditPostCommand = require('../../structures/command/RandomRedditPostCommand.js')
```
Comment (bolsomito) [inline src/commands/reddit/oddlysatisfying.js]: ```const { RandomRedditPostCommand } = require('../../')```

File: src/commands/reddit/softwaregore.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+const RandomRedditPostCommand = require('../../structures/command/RandomRedditPostCommand.js')
```
Comment (bolsomito) [inline src/commands/reddit/softwaregore.js]: ```const { RandomRedditPostCommand } = require('../../')```

File: src/commands/reddit/tesla.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+const RandomRedditPostCommand = require('../../structures/command/RandomRedditPostCommand.js')
```
Comment (bolsomito) [inline src/commands/reddit/tesla.js]: ```const { RandomRedditPostCommand } = require('../../')```

File: src/commands/reddit/thinking.js [hunk 7]
```diff
@@ -0,0 +1,12 @@
+const RandomRedditImageCommand = require('../../structures/command/RandomRedditPostCommand.js')
```
Comment (bolsomito) [inline src/commands/reddit/thinking.js]: ```const { RandomRedditImageCommand } = require('../../')```

File: src/commands/reddit/aww.js (hunk 1 above)
Comment (bolsomito) [reply src/commands/reddit/aww.js]: also, are you referencing this as RandomRedditPostCommand?

File: src/commands/reddit/thinking.js (hunk 7 above)
Comment (bolsomito) [reply src/commands/reddit/thinking.js]: also, are you referencing this as RandomRedditPostCommand?

**Comment by pedrofracassi** [comment]
@bolsomito doing it the way you want doesn't work as it causes a require loop.

**Comment by pedrofracassi** [comment]
@davipatury is there a better way of doing the requires? The size is bothering me too.

**Comment by davipatury** [comment]
Maybe there is, but probrably there'll be some problems is the recursiveness in the `src/index.js`. :frowning_face:

**Pull request merged**